### PR TITLE
Add FreeBSD support to installer

### PR DESCRIPTION
**Pull request by rickvanderzwet**
Configuration directory /etc is reserved for system configuration files.
/usr/local/etc is reserved for local (user) installed configuation files.

### CODE REVIEW

**Comment by bilde2910** [comment]
Interesting; I wasn't aware that FreeBSD didn't follow the same directory setup as Linux.

**Comment by rickvanderzwet** [comment]
> 
> 
> Interesting; I wasn't aware that FreeBSD didn't follow the same directory setup as Linux.

File: install.sh [hunk 1]
```diff
@@ -14,11 +14,21 @@ hauk_help() {
...
     echo ""
     echo -e "\033[1mOptions:\033[0m"
-    echo -e "\033[91m-c\t\033[0mInstall config file to /etc/hauk"
+    echo -e "\033[91m-c\t\033[0mInstall config file to $config"
     echo -e "\033[91m-f\t\033[0mOverwrite without asking"
     exit 1
 }
 
+# Centralized configuration
+if [ "X$(uname -s)" = "XFreeBSD" ]; then
```
[the remark this answers is not in the document]
Comment (rickvanderzwet) [reply install.sh]: > Just out of curiosity, what's the purpose of the X before the `uname` call here? On my Linux system, it works fine without the X, but I'm not sure if I'm just unaware of some reason it _should_ be there.

You are right is it no longer required, [old habbits](https://stackoverflow.com/a/6853353) die slow.

**Comment by rickvanderzwet** [comment]
Whoops sorry, wrong button

**Comment by bilde2910** [comment]
Thanks, merging :+1:

**Comment by rickvanderzwet** [comment]
> Interesting; I wasn't aware that FreeBSD didn't follow the same directory setup as Linux.

Also see man [hier(7)](https://www.freebsd.org/cgi/man.cgi?query=hier&apropos=0&sektion=7) on FreeBSD pages. Quite usefull feature to be honest, one could always delete the complete `/usr/local` directory, reboot and the system is back to 'vanilla'.

**Pull request merged**